tests/basic/run.sh

```diff
@@ -31,4 +32,16 @@ run_dumpling --where "a >= 3 and a <= 9" -f "$DB_NAME.$TABLE_NAME"
 
 actual=$(grep -w "(.*)" ${DUMPLING_OUTPUT_DIR}/${DB_NAME}.${TABLE_NAME}.000000000.sql | cut -c2-2)
 expected=$(seq 3 9)
+echo "expected ${expected}, actual ${actual}"
 [ "$actual" = "$expected" ]
+
+export DUMPLING_TEST_PORT=4000
+# Test for --sql option.
+run_sql "drop database if exists \`$DB_NAME\`;"
+run_sql "create database \`$DB_NAME\`;"
+run_sql "create sequence \`$DB_NAME\`.\`SEQUENCE_NAME\` increment by 1;"
+run_dumpling --sql "select nextval(\`$DB_NAME\`.\`SEQUENCE_NAME\`)"
+
+actual=$(grep -w "(.*)[,|;]" ${DUMPLING_OUTPUT_DIR}/result.000000000.sql | cut -c2-2)
+echo "expected 1, actual ${actual}"
+[ "$actual" = 1 ]
```

Comment:
please also `run_sql 'select nextval()'` and check the result is 2.

Reply:
Updated.